src/main/java/net/mcreator/workspace/misc/WorkspaceInfo.java

```diff
@@ -117,11 +116,14 @@ public List<ModElement> getElementsOfType(ModElementType<?> type) {
 		}
 	}
 
-	public List<ModElement> getRecipesOfType(String typestring) {
+	public List<GeneratableElement> getRawElementsOfType(String typestring) {

```

Comment:
I would rename this to getModElements(type) for the sake of shorter lines in definition files